[matplotplusplus] .cmake file not found by CMake

**Describe the bug**
CMake can't find matplotplusplus' .cmake file during configuration after installation with vcpkg.

**Environment**
- OS: Ubuntu 18.04
- Compiler: GCC 10.3.0
- cmake 3.20.1

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install matplotplusplus
2. Add to CMakeLists.txt: `find_package(Matplot++ CONFIG REQUIRED)`
3. Building with `cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_TOOLCHAIN_FILE=/home/xander/dev/vcpkg/scripts/buildsystems/vcpkg.cmake .` produces this error:
```
CMake Error at /home/xander/dev/vcpkg/scripts/buildsystems/vcpkg.cmake:764 (_find_package):
  Could not find a package configuration file provided by "Matplot++" with
  any of the following names:

    Matplot++Config.cmake
    matplot++-config.cmake

  Add the installation prefix of "Matplot++" to CMAKE_PREFIX_PATH or set
  "Matplot++_DIR" to a directory containing one of the above files.  If
  "Matplot++" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:38 (find_package)


-- Configuring incomplete, errors occurred!
```

**Expected behavior**
Like other vcpkg installed libraries, I would expect this to be found without additional configuration.

**Additional context**
All of my other vcpkg-installed libraries are successfully found by cmake, I'm not sure what's different about this one.

I see one of the two named .cmake files exists on the system:
```
$ find / -name matplot++-config.cmake -print 2>/dev/null
/home/xander/dev/vcpkg/buildtrees/matplotplusplus/x64-linux-dbg/matplot++-config.cmake
/home/xander/dev/vcpkg/buildtrees/matplotplusplus/x64-linux-rel/matplot++-config.cmake
$ find / -name Matplot++Config.cmake -print 2>/dev/null
```
Any thoughts on why it exists only in the build tree and is not found by cmake? I would expect the .cmake file to end up in `vcpkg/installed/x64-linux/`, right?

--- a/ports/matplotplusplus/portfile.cmake
+++ b/ports/matplotplusplus/portfile.cmake
@@ -50,10 +50,10 @@ vcpkg_copy_pdbs()
 
 file(RENAME
     "${CURRENT_PACKAGES_DIR}/lib/cmake/Matplot++/matplot++-config.cmake"
-    "${CURRENT_PACKAGES_DIR}/lib/cmake/Matplot++/Matplot++-config.cmake"
+    "${CURRENT_PACKAGES_DIR}/lib/cmake/Matplot++/Matplot++Config.cmake"
 )
 
-vcpkg_cmake_config_fixup(PACKAGE_NAME Matplot++ CONFIG_PATH lib/cmake/Matplot++)
+vcpkg_cmake_config_fixup(PACKAGE_NAME matplot++ CONFIG_PATH lib/cmake/Matplot++)
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/debug/share")
 

--- a/ports/matplotplusplus/portfile.cmake
+++ b/ports/matplotplusplus/portfile.cmake
@@ -48,11 +48,6 @@ vcpkg_cmake_install()
 
 vcpkg_copy_pdbs()
 
-file(RENAME
-    "${CURRENT_PACKAGES_DIR}/lib/cmake/Matplot++/matplot++-config.cmake"
-    "${CURRENT_PACKAGES_DIR}/lib/cmake/Matplot++/Matplot++Config.cmake"
-)
-
 vcpkg_cmake_config_fixup(PACKAGE_NAME matplot++ CONFIG_PATH lib/cmake/Matplot++)
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/debug/share")

--- a/ports/matplotplusplus/vcpkg.json
+++ b/ports/matplotplusplus/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "matplotplusplus",
   "version-date": "2021-04-11",
-  "port-version": 3,
+  "port-version": 4,
   "description": "A C++ graphics library for data visualization",
   "homepage": "https://alandefreitas.github.io/matplotplusplus/",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3922,7 +3922,7 @@
     },
     "matplotplusplus": {
       "baseline": "2021-04-11",
-      "port-version": 3
+      "port-version": 4
     },
     "matroska": {
       "baseline": "1.6.2",

--- a/versions/m-/matplotplusplus.json
+++ b/versions/m-/matplotplusplus.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ea0bc7cf9fc9b903206f903da8d915930773862d",
+      "version-date": "2021-04-11",
+      "port-version": 4
+    },
+    {
       "git-tree": "e60b72bfb17b3263a7ee1e056af221991f5c7d69",
       "version-date": "2021-04-11",
       "port-version": 3

--- a/versions/m-/matplotplusplus.json
+++ b/versions/m-/matplotplusplus.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "ea0bc7cf9fc9b903206f903da8d915930773862d",
+      "git-tree": "ee89d0f23b11e362fb9e65670377c47b35ffea6c",
       "version-date": "2021-04-11",
       "port-version": 4
     },

